frontend/src/concepts/pipelines/topology/core/PipelineTopology.tsx

```diff
@@ -12,14 +12,16 @@ type PipelineTopologyProps = {
   selectedIds?: string[];
   onSelectionChange?: (selectionIds: string[]) => void;
   nodes: PipelineNodeModel[];
+  disableSelection?: boolean;
 };
 
 const PipelineTopology: React.FC<PipelineTopologyProps> = ({
   nodes,
   selectedIds,
   onSelectionChange,
+  disableSelection,
 }) => {
-  const controller = useTopologyController('g1');
+  const controller = useTopologyController('g1', disableSelection);
```

Comment:
Is there a reason we want to do this? The Details page should probably have the Pipeline sidebar & Topology, but some details as if it was a run (input params). 

* Pipelines are definition
* Runs are execution
* Jobs are definitions that create execution -- so we wanna combine as much of the two flows as possible

Reply:
Oh boy... I just read my ticket and I said disable selection haha... @yannnz @kywalker-rh I'm wondering why this is a good idea. I had it in my head that we wanted selection to be akin to Pipeline Details 🤔

Reply:
any update on this? Should i add back the selection to include:
> Pipelines are definition
Runs are execution
Jobs are definitions that create execution -- so we wanna combine as much of the two flows as possible

Reply:
It seems to have been missed by Yan -- might need to wait until next week to get an answer.

Reply:
> It seems to have been missed by Yan -- might need to wait until next week to get an answer.

Hmm, I am sorry I missed this mention somehow. So what is needed here? Are we saying we should allow the user to select components and seeing details of Input parameters of the selected on in the right side drawer?

Reply:
@yannnz do you have an opinion on disabling selection of topography nodes in the pipeline details view for scheduled runs?

Reply:
It is also how the triggered run designed nowadays regardless of the data difference - nodes info in the right side panel, and run details in the bottom drawer,  so I think it makes sense to have the same pattern for the scheduled runs. But I am not sure what is the run input parameters? Is it similar to the run output parameter? @Gkrumbach07 @andrewballantyne

Reply:
Yes, the data for the `run input parameters` are in the `run output` tab's yaml on the bottom drawer. We are surfacing them in https://github.com/opendatahub-io/odh-dashboard/issues/1245

The Input Params are the items you put at the bottom of the Create Run page to supply with inputs to the run